.github/actions/pre-commit/action.yml

```diff
@@ -11,7 +20,15 @@ runs:
     - uses: actions/checkout@v3
     - name: Set up Python ${{ inputs.python-version }}
       uses: actions/setup-python@v3
+    - name: build extra_args
+      shell: bash
+      env:
+        EXTRA_ARGS: ''
+      run: |
+        [ ${{ inputs.pre-commit-all-files }} ] && EXTRA_ARGS='--all-files'
+        EXTRA_ARGS="${EXTRA_ARGS} ${{ inputs.pre-commit_args }}"
+        echo "EXTRA_ARGS="${EXTRA_ARGS}" >> $GITHUB_ENV"
+    - uses: pre-commit/action@v2.0.3
```

Comment:
I see this action has a [v3.0.0](https://github.com/pre-commit/action/releases/tag/v3.0.0) version available, but I don't think it makes any difference for us here at all